### PR TITLE
OPENEUROPA-2606: Add optional link field to Social media follow paragraph.

### PR DESCRIPTION
**Pull request by 22Alexandra**
### Description

Add optional link field to Social media follow paragraph.

### Change log

- Added: optional link field
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
drush updb

```

### CODE REVIEW

File: tests/features/social-media-follow.feature [hunk 1]
```diff
@@ -31,11 +31,14 @@ Feature: Social media follow paragraph.
     And I fill in the 3rd "URL" field with "http://twitter.com"
     And I fill in the 3rd "Link text" with "2nd Twitter"
     And I select "Twitter" from the 3rd "Link type"
+    And I fill in the 4th "URL" field with "https://europa.eu/european-union/contact/social-networks_en"
```
Comment (imanoleguskiza) [inline tests/features/social-media-follow.feature]: I'm not completely happy with this step. As it is, it seems that we are filling another social link and indeed when we check the rendered values nothing implies it isn't another social link. Could we make sure we are filling the correct link by using the same format as in oe_content? (see https://github.com/openeuropa/oe_content/blob/EPIC-Event/tests/features/event.feature#L197)

File: oe_paragraphs.post_update.php [hunk 2]
```diff
@@ -83,3 +85,43 @@ function oe_paragraphs_post_update_10003(array &$sandbox): void {
...
 }
+
+/**
+ * Add optional link field to Social media follow paragraph.
+ */
+function oe_paragraphs_post_update_10004(array &$sandbox): void {
+  \Drupal::service('module_installer')->install(['link']);
+
+  $storage = new FileStorage(drupal_get_path('module', 'oe_paragraphs') . '/config/post_updates/10004');
+  $field_config = [
```
Comment (nagyad) [inline oe_paragraphs.post_update.php]: Make it prular.

File: oe_paragraphs.post_update.php [hunk 3]
```diff
@@ -83,3 +85,43 @@ function oe_paragraphs_post_update_10003(array &$sandbox): void {
...
+    'field.storage.paragraph.field_oe_social_media_see_more',
+    'field.field.paragraph.oe_social_media_follow.field_oe_social_media_see_more',
+  ];
+  $config_manager = \Drupal::service('config.manager');
+  $entity_manager = \Drupal::entityTypeManager();
+  foreach ($field_config as $config) {
```
Comment (nagyad) [inline oe_paragraphs.post_update.php]: Please rename config to field_config.

File: oe_paragraphs.post_update.php [hunk 4]
```diff
@@ -83,3 +85,43 @@ function oe_paragraphs_post_update_10003(array &$sandbox): void {
...
+    $config_record = $storage->read($config);
+    $entity_type = $config_manager->getEntityTypeIdByName($config);
```
Comment (nagyad) [inline oe_paragraphs.post_update.php]: Do we need to add check here if we have anything returned?

**Pull request merged**